src/test/java/io/github/ust/mico/kafkafaasconnector/MessageListenerTests.java

```diff
@@ -191,5 +270,75 @@ public void testRouteHistory() {
         consumer.close();
     }
 
+    @Test
+    public void testFilterOutCheck() {
```

Comment:
Add javadoc to describe tests with a little more detail than the method name allows